core-galleon-pack/src/main/resources/layers/standalone/elytron/layer-spec.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="elytron">   
+    <dependencies>
+        <layer name="base-server"/> 
+    </dependencies>
+    <!-- elytron layer comes with access-control -->
+    <feature-group name="access-control"/>
```

Comment:
This doesn't sound right. There shouldn't be anything special about elytron that adds a need for management RBAC configuration.

Reply:
There is no RBAC layer, RBAC seems to make sense when security is configured. That is why elytron brings it in.